ensime_shared/vim_commands.py

```diff
@@ -0,0 +1,44 @@
+commands = {
```

Comment:
Just noticed, FWIW, the same thing is already basically done with the `feedback` dict. If we keep the extraction of `commands`, maybe it could at least go in `config.py` alongside `feedback` just to reduce proliferation of files.